scripts/hotfixes/kdump/README.md

```diff
@@ -0,0 +1,12 @@
+# Kernel Dump Hotfix
+
+This hotfix comes from (`CASMTRIAGE-3591`)[https://jira-pro.its.hpecorp.net:8443/browse/CASMTRIAGE-3591].
```

Comment:
Agreed. I don't think we should link to internal Jiras in the customer-facing docs, since they won't be able to follow the link. If we just want to provide a breadcrumb for internal folks, I think that makes more sense as a comment in the script.

Reply:
Moved to script.